iosApp/iosApp/Pages/Map/HomeMapViewLayerExtension.swift

```diff
@@ -60,13 +60,13 @@ extension HomeMapView {
 
     func refreshMap() {
         if let layerManager = mapVM.layerManager {
+            updateGlobalMapDataSources()
```

Comment:
At one point I was getting warning messages about the sources for layers not being present. Updating the sources above adding the layers fixed the issue. This may be rendered a moot change by switching `addLayers` to `addPersistedLayers` below, but seemed worth keeping around.